echopype/tests/test_convert_ad2cp.py

```diff
@@ -112,7 +103,9 @@ def test_raw_output():
                 assert np.isclose(
                     echodata.beam_complex["echosounder_raw_transmit_samples_r"]
                     .sel(
-                        ping_time=echodata.beam_complex["time_echosounder_raw_transmit"]
+                        ping_time=echodata.beam_complex[
+                            "ping_time_echosounder_raw_transmit"
+                        ]
```

Comment:
For this and the next few assert, what's the actual difference between the ping_time numbers? I am wondering about np.isclose as it has a default tolerance.

Reply:
The ping_time numbers should be the same I believe, but the sample values themselves are slightly off:
![image](https://user-images.githubusercontent.com/49664304/119198669-92e2ae00-ba3e-11eb-8168-483371faeac4.png)

Thanks for letting me know about allclose!